src/components/course/data/utils.jsx

```diff
@@ -209,10 +211,7 @@ export function shouldUpgradeUserEnrollment({
   enrollmentUrl,
 }) {
   const isAuditEnrollment = userEnrollment?.mode === COURSE_MODES_MAP.AUDIT;
-  if (isAuditEnrollment && subscriptionLicense && enrollmentUrl) {
-    return true;
-  }
-  return false;
+  return isAuditEnrollment && subscriptionLicense && enrollmentUrl;
```

Comment:
nit: Is there a chance this won't return a true boolean value? Should we wrap with `!!`?

```return !!(isAuditEnrollment && subscriptionLicense && enrollmentUrl)```?

Reply:
good call out, yeah we'd want this to be a boolean.